qtt/measurements/scans.py

```diff
@@ -494,6 +494,21 @@ def scan1Dfast(station, scanjob, location=None, liveplotwindow=None, delete=True
     t0 = time.time()
     wait_time_startscan = scanjob.get('wait_time_startscan', 0)
 
+    try:
+        isfpga = isinstance(
+            minstrhandle, qtt.instrument_drivers.FPGA_ave.FPGA_ave)
+    except:
+        isfpga = False
+    try:
```

Comment:
Why is the `isinstance` in a `try` and `except` block? If the `minstrhandle` is not of the correct type it will return False. Am I maybe missing something?

Reply:
`qtt.instrument_drivers.FPGA_ave.FPGA_ave` cannot be imported on a system without an FPGA, therefore we used a `try`-`catch`. Better would have been to catch the specific exception. @CumulonimbusCalvus